dspace-api/src/main/java/org/dspace/submit/step/DescribeStep.java

```diff
@@ -406,24 +414,7 @@ else if (getErrorFields(request) != null && getErrorFields(request).size() > 0)
     public int getNumberOfPages(HttpServletRequest request,
             SubmissionInfo subInfo) throws ServletException
     {
-        // by default, use the "default" collection handle
-        String collectionHandle = DCInputsReader.DEFAULT_COLLECTION;
-
-        if (subInfo.getSubmissionItem() != null)
-        {
-            collectionHandle = subInfo.getSubmissionItem().getCollection()
-                    .getHandle();
-        }
-
-        // get number of input pages (i.e. "Describe" pages)
-        try
-        {
-            return getInputsReader().getNumberInputPages(collectionHandle);
-        }
-        catch (DCInputsReaderException e)
-        {
-            throw new ServletException(e);
-        }
+        return 1;
```

Comment:
If it is always 1, do we still need this method/class?

Reply:
Yes by now the concept of page in the step should be deprecated. One step one page.

Reply:
Then I would remove the `getNumberOfPages` method to avoid confusion.

Reply:
Agreed, if it's always `1`, then this method probably should be removed

Reply:
my suggestion for the first phase was to just keep the interface of the dspace-api untouched as much as possible to reduce the number of classes touched by the PR. We will need for sure to review the Step interface as soon as we start to deal with the "update" endpoint of workspaceitem